tests/test_utils.py

```diff
@@ -0,0 +1,74 @@
+"""Tests for utility functions."""
+
+import os
+
+import pytest
+
+from ruff_cgx import get_ruff_command, reset_ruff_command, set_ruff_command
+from ruff_cgx.utils import run_ruff_check, run_ruff_format
+
+
+@pytest.fixture(autouse=True)
+def reset_config():
+    """Reset ruff command configuration before and after each test."""
+    reset_ruff_command()
```

Comment:
Your test setup depends on your code under test :)

Reply:
Ai, nasty! 🤦

Reply:
At first glance it seemed alright, but it is indeed better to not do it this way!

Reply:
Well. You have to have the state _somewhere_. :) It can be a class member or a module member... who cares. What's important is that your test will fail if the implementation breaks, or if the state moves elsewhere, it will also break! :+1: